site/content/en/docs/Advanced/allocator-service.md

```diff
@@ -30,9 +30,16 @@ agones-allocator            LoadBalancer   10.55.251.73    <b>34.82.195.204</b>
 
 ## Server TLS certificate
 
-If the `agones-allocator` service is installed as a `LoadBalancer` [using a static IP]({{< relref "/docs/Installation/Install Agones/helm.md#reserved-allocator-load-balancer-ip" >}}), a valid self-signed server TLS certificate is generated using the IP provided. Otherwise, the server TLS certificate should be replaced.
+If the `agones-allocator` service is installed as a `LoadBalancer` [using a static IP]({{< relref "/docs/Installation/Install Agones/helm.md#reserved-allocator-load-balancer-ip" >}}), a valid self-signed server TLS certificate is generated using the IP provided. Otherwise, the server TLS certificate should be replaced. For a quick start, run helm upgrade again, while setting `agones.allocator.http.loadBalancerIP`.
 
-Replace the default server TLS certificate with a certificate with CN and subjectAltName. There are multiple approaches to generate a certificate. Agones recommends using [cert-manager.io](https://cert-manager.io/) solution for cluster level certificate management.
+```bash
+EXTERNAL_IP=`kubectl get services agones-allocator -n agones-system -o jsonpath='{.status.loadBalancer.ingress[0].ip}'`
```

Comment:
Nit: We should be using $() over \`\`, even though `` does still work.

Context:
https://stackoverflow.com/a/4708569
http://mywiki.wooledge.org/BashFAQ/082 - Why is $(...) preferred over \`...\` (backticks)?

At some point we should go through and fixup the other docs.